test/module/irohad/network/block_loader_test.cpp

```diff
@@ -50,14 +47,20 @@ class BlockLoaderTest : public testing::Test {
 
     grpc::ServerBuilder builder;
     int port = 0;
-    builder.AddListeningPort(peer.address, grpc::InsecureServerCredentials(),
+    builder.AddListeningPort(address, grpc::InsecureServerCredentials(),
                              &port);
     builder.RegisterService(service.get());
     server = builder.BuildAndStart();
+
+    peer = Peer();
+    peer.address = "0.0.0.0:" + std::to_string(port);
```

Comment:
Why it's `0.0.0.0` by the way? I think it should be set to localhost

Reply:
It was in all tests, so I decided not to change it

Reply:
Sonarcube issue fixed

Reply:
Yeah, I know. Question is for @lebdron

Reply:
https://en.wikipedia.org/wiki/0.0.0.0
>A way to specify "any IPv4 address at all". It is used in this way when configuring servers (i.e. when binding listening sockets). 

This seems more applicable than a special address like 127.0.0.1

Reply:
"Address at all" basically means that it binds to **every** available address. That means host is going to be publicly available in the all related networks. Don't really sure it's secure (nevertheless tests are quite fast)